tests/tdastro/astro_utils/test_passbands.py

```diff
@@ -513,7 +513,7 @@ def test_passband_fluxes_to_bandflux_mult_samples(passbands_dir, tmp_path):
 
 
 def test_passband_wrapped_from_physical_source(passbands_dir, tmp_path):
-    """Test get_band_fluxes, PhysicalModel's wrapped version of Passband's fluxes_to_bandflux.."""
+    """Test evaluate_band_fluxes, PhysicalModel's wrapped version of Passband's fluxes_to_bandflux.."""
```

Comment:
```suggestion
    """Test evaluate_band_fluxes, PhysicalModel's wrapped version of Passband's fluxes_to_bandflux."""
```
Just happened to notice the double period at the end